.editorconfig

```
@@ -0,0 +1,26 @@
+# http://editorconfig.org/
+
+root = true
+
+[*]
+end_of_line = lf
+insert_final_newline = true
+charset = utf-8
+trim_trailing_whitespace = true
+
+[*.go]
+indent_style = tab
+
+[*.json]
+indent_size = 2
+
+[Makefile,*.mk]
+indent_style = tab
+
+[*.md]
+indent_style = space
+indent_size = 4
```

Would you please add 
```suggestion
indent_size = 4
trim_trailing_whitespace = false
```

---

Happy to do so -- why is this required?

---

You can create line breaks in markdown with a double white space at the end of the line. `trim_trailing_whitespace = true` remove them, which might be unexpecting.